hoprd/rest-api/src/lib.rs

```diff
@@ -816,7 +814,7 @@ mod account {
         currency: BalanceType,
         #[serde_as(as = "DisplayFromStr")]
         #[schema(value_type = String)]
-        amount: U256,
+        amount: String,
```

Comment:
It should be a u128 value and then convert U256::from(u128_value). String is too wide and can fail for too many reasons.

Reply:
In the API, all amount representation are in string, only this one was as a U256. It was changed for coherency.

Reply:
Then I'd say those strings are incorrect.
It should be a number, as big as possible, to fit into U256, ideally convertible from U256 in its Display format.

Reply:
Can we do the change to u256 in the next time we break the API to avoid CT, uHTTP, and NodeAdmin to change their implementation ?